deepchem/models/tests/test_gnn.py

```diff
@@ -0,0 +1,33 @@
+import os
+import pytest
+import numpy as np
+import deepchem as dc
+from deepchem.feat.molecule_featurizers import SNAPFeaturizer
+
+
+def get_regression_dataset():
+    np.random.seed(123)
+    featurizer = SNAPFeaturizer()
+    dir = os.path.dirname(os.path.abspath(__file__))
+
+    input_file = os.path.join(dir, 'assets/example_regression.csv')
+    loader = dc.data.CSVLoader(tasks=["outcome"],
+                               feature_field="smiles",
+                               featurizer=featurizer)
+    dataset = loader.create_dataset(input_file)
+    metric = dc.metrics.Metric(dc.metrics.mean_absolute_error,
+                               mode="regression")
+
+    return dataset, metric
+
+
+@pytest.mark.torch
+def test_GNN_edge_pred():
```

Comment:
We will need the usual overfit, save/reload tests. Will those follow in subsequent PRs?

Reply:
test_GNN_edge_pred() is my attempt at a low resource overfit test, just checking if the loss is going down after between 5 epochs. Hard to determine what 'overfit' means for an unsupervised task. I have added a save/reload test. There are more tests in the coming PRs.